<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>5</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Botify</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_open">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>221</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Open your SC/BLA Export File</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelFilename">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>240</y>
            <width>811</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Selected file</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_query">
          <property name="geometry">
           <rect>
            <x>680</x>
            <y>50</y>
            <width>141</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Get Top Query Parameter</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>651</width>
            <height>51</height>
           </rect>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Go to &lt;span style=&quot; font-weight:700;&quot;&gt;SiteCrawler&lt;/span&gt; or &lt;span style=&quot; font-weight:700;&quot;&gt;Log Analyzer&lt;/span&gt; and export URLs containing query parameters&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Suggested filter: URL Query match regex .+&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>361</width>
            <height>51</height>
           </rect>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Go to &lt;span style=&quot; font-weight:700;&quot;&gt;Keywords Explorer&lt;/span&gt; and choose Keyword, Impressions, and Clicks as columns (the default view)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_keyword">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>150</y>
            <width>151</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Get Top unique Keyword</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_csv">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>150</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>In the Window</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_cloud">
          <property name="geometry">
           <rect>
            <x>670</x>
            <y>150</y>
            <width>161</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>TagClouds (limited to 100)</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_language">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>180</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>english</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>french</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>german</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>italian</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>spanish</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>180</y>
            <width>191</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Language for removing stop words</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_open_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>221</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Open your RK Export File</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>PageWorkers</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>URL</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditURL"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_GoURL">
            <property name="text">
             <string>Go</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Sitemap</string>
         </attribute>
         <widget class="QLineEdit" name="lineEdit_sitemap">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>150</y>
            <width>651</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>131</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Sitemaps URL</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_sitemap">
          <property name="geometry">
           <rect>
            <x>760</x>
            <y>150</y>
            <width>75</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Retrieve</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>801</width>
            <height>71</height>
           </rect>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Extract all URLs present in the sitemap.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If hreflang is present add columns with the link between the url and language&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Robots.txt</string>
         </attribute>
         <widget class="QPlainTextEdit" name="plainTextEdit_rules">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>50</y>
            <width>751</width>
            <height>101</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Lucida Console</family>
           </font>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_ua">
          <property name="geometry">
           <rect>
            <x>490</x>
            <y>10</y>
            <width>331</width>
            <height>26</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:137.0) Gecko/20100101 Firefox/137.0)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mozilla/5.0 AppleWebKit/537.36 (KHTML, like Gecko; compatible; Googlebot/2.1; +http://www.google.com/bot.html) Chrome/W.X.Y.Z Safari/537.36</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mozilla/5.0 (compatible; botify; http://botify.com)</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>10</y>
            <width>49</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>UA</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_retrieve_robots">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>10</y>
            <width>75</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Retrieve</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_rob">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>10</y>
            <width>301</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Robots.txt URL</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit_urls">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>160</y>
            <width>751</width>
            <height>101</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>49</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Rules</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>190</y>
            <width>49</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>URLs</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_check_robots">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>260</y>
            <width>751</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Check URLs</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>JSON</string>
         </attribute>
         <widget class="QLineEdit" name="lineEdit_burl">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>260</y>
            <width>541</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>260</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Botify's URL with filter</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_json">
          <property name="geometry">
           <rect>
            <x>720</x>
            <y>260</y>
            <width>75</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Get JSON</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>100</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>JSON input</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_json_validate">
          <property name="geometry">
           <rect>
            <x>740</x>
            <y>90</y>
            <width>91</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>Validate 
 and 
Make beautiful</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit_json">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>20</y>
            <width>631</width>
            <height>201</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>API</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowserOutput"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
